internal/coreinternal/timeutils/parser.go

```diff
@@ -178,5 +178,9 @@ func ValidateLocale(locale string) error {
 	return fmt.Errorf("invalid locale '%s': %w", locale, err)
 }
 
+func GetStrptimeNativeSubstitutes(format string) map[string]string {
```

Comment:
```suggestion
// GetStrptimeNativeSubstitutes analyzes the provided format string and returns a map
// where each key is a Go native layout element (as used in time.Format) found in the
// format, and each value is the corresponding ctime-like directive.
func GetStrptimeNativeSubstitutes(format string) map[string]string {
```

Reply:
added the comment with the latest commit